src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -393,9 +398,12 @@ export default {
 		canBeDemoted() {
 			return this.canModerate
 				&& [PARTICIPANT.TYPE.MODERATOR, PARTICIPANT.TYPE.GUEST_MODERATOR].indexOf(this.participantType) !== -1
+				&& !this.isGroup
```

Comment:
Instead of modifying `canBeDemoted` and `canBePromoted` I think that it would be better to add the condition to `canModerate` (and rename it to `canBeModerated`, as the current name is misleading).

Reply:
But then you couldn't remove the groups anymore as that also checks "canModerate".

I agree on the misleading name and will fix it, but the isGroup check needs to be on the promote/demote